core/storage/suggestion/gae_models_test.py

```diff
@@ -330,6 +330,43 @@ def test_query_suggestions_by_language(self) -> None:
             len(suggestion_models.GeneralSuggestionModel.query_suggestions(
                 queries)), 1)
 
+    def test_get_in_review_translation_suggestions(self) -> None:
+        suggestion_models.GeneralSuggestionModel.create(
+            feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+            feconf.ENTITY_TYPE_EXPLORATION,
+            'exp1', self.target_version_at_submission,
+            suggestion_models.STATUS_IN_REVIEW, 'author_3',
+            'reviewer_2', self.change_cmd, self.score_category,
+            'exploration.exp1.thread_6', self.translation_language_code)
+        suggestion_models.GeneralSuggestionModel.create(
+            feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT,
+            feconf.ENTITY_TYPE_EXPLORATION,
+            'exp1', self.target_version_at_submission,
+            suggestion_models.STATUS_IN_REVIEW, 'author_4',
+            'reviewer_2', self.change_cmd, self.score_category,
+            'exploration.exp1.thread_7', self.translation_language_code)
+
+        suggestions = (
+            suggestion_models.GeneralSuggestionModel
+            .get_in_review_translation_suggestions(
+                'exp1', [self.translation_language_code]))
+
+        self.assertEqual(len(suggestions), 2)
```

Comment:
For this to be a proper test, I think you'll need to add some suggestions that shouldn't be caught by the query?

Reply:
Some non-translation suggestions are created in the test setup, so I added translation suggestions not in review to test the suggestion status.